src/main/webapp/chart.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
```

Comment:
"Indent by 2 spaces at a time."
You should re-format your code using [this guilde style](https://github.com/google/styleguide)